src/generators/starter-app/index.js

```diff
@@ -8,7 +8,16 @@ const StarterAppMixin = subclass =>
   class extends BuildingMixin(TestingMixin(LintingMixin(subclass))) {
     async execute() {
       // before super to also affect the Mixin it applies
-      const { tagName, className } = await askTagInfo();
+      let tagName = '';
+      let className = '';
+
+      if(process.argv[3]) {
```

Comment:
how about adding this to the askTagInfo directly so we could execute every generator with these predefined values?